src/index.js

```diff
@@ -62,4 +62,57 @@ program
       });
     });
 
+
+program
+  .command('portfolio [fileLocation]')
+  .option('-m, --market [market]')
+  .action((fileLocation,options) => {
+      const fullPath = process.cwd()+'/'+fileLocation;
+      const market = options.market || 'cmc';
+      let fs = require('fs');
+      fs.readFile(fullPath, 'utf8', function (err, data) {
+          if (err) throw err; 
+          const obj = JSON.parse(data);
+          let keys = Object.keys(obj);
+          let proms = [];
+          let portfolioData = {};
+          keys.forEach(function(key) {
+              portfolioData[key.toUpperCase()] = obj[key];
+              proms.push(getCurrecyData(market.toLowerCase(),key));
+          }, this);
+          Promise.all(proms)
+          .then(res =>{
+                showPortfolio(res,portfolioData);
+          }, err => {
+             console.log('Errored '+err);
+          });
+      });
+  });
+
+  function getCurrecyData(market,currency){
+      if(market == 'gdax')
+          return  GDAX.getCurrency(currency);
+      else if(market == 'kraken')
+        return  Kraken.getCurrency(currency);
+      else if(market == 'cmc')
+        return  CoinMarketCap.getCurrency(currency);
+      else
+        return null;
+  }
+
+ function showPortfolio(res,obj){
+  console.log('--------------------------------------------------------------------------------------------');
+  console.log(`Symbol   Price     Acct. Val     %Change`);
+  console.log('--------------------------------------------------------------------------------------------');
+  let total = 0;
+  res.forEach(function(currency) {
+
+      let currentVal =  Helpers.round( obj[currency.symbol]*currency.price_usd);
+      total+=Number(currentVal);
+      console.log(`${currency.symbol}    $${Helpers.round(currency.price_usd)}    $${currentVal}      $${currency.percent_change_24h}%`);
```

Comment:
Looks like you have a dollar sign in front of the percentage. Might want to remove that:

`${currency.percent_change_24h}%`